Source/Core/Core/IOS/FS/HostBackend/FS.h

```diff
@@ -0,0 +1,84 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <array>
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "Common/CommonTypes.h"
+#include "Common/File.h"
+#include "Core/IOS/FS/FileSystem.h"
+
+namespace IOS::HLE::FS
+{
+/// Backend that uses the host file system as backend.
+///
+/// Ignores metadata like permissions, attributes and various checks and also
+/// sometimes returns wrong information, but kept for backwards compatibility.
```

Comment:
This claims "backwards compatibility" but appears to be the only implementation of IOS::HLE::FS::Filesystem, correct?